### PR TITLE
New nav design

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #731 , #786

### CODE REVIEW

File: components/navbar/navbar.jsx [hunk 1]
```diff
@@ -36,23 +43,17 @@ class NavBar extends React.Component {
...
-  renderName() {
+  renderName(classes) {
     // don't show anything until we verify this user's loggedin status
     if (user.loggedin === undefined) return null;
 
     let link = <a href={user.getLoginURL(utility.getCurrentURL())} onClick={(event) => this.handleSignInBtnClick(event)}>Signup / Signin</a>;
 
     if (user.loggedin) {
-      link = <NavLink to="/profile/me">Hi, {user.name}</NavLink>;
+      link = <NavLink to="/profile/me">Hi {user.name}</NavLink>;
```
Comment (gvn) [inline components/navbar/navbar.jsx]: The comma is more grammatically correct, but `¯\_(ツ)_/¯`.

File: components/navbar/navbar.scss [hunk 2]
```diff
@@ -1,99 +1,129 @@
...
-    position: static;
-    align-self: center;
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.9);
+    z-index: 1000000;
```
Comment (gvn) [inline components/navbar/navbar.scss]: Lolz.

**Comment by gvn** [comment]
When I click my name in the nav it doesn't close the popover.

File: components/navbar/navbar.jsx (hunk 1 above)
Comment (mmmavis) [reply components/navbar/navbar.jsx]: I like keeping `,` but https://github.com/mozilla/network-pulse/issues/786 `¯\_(ツ)_/¯` 😂

Comment (mmmavis) [reply components/navbar/navbar.jsx]: I'll remove "Hi" as well

**Comment by mmmavis** [comment]
🎉  pushing it to staging now so designers can QA 💃 :man_dancing:

**Pull request merged**